### PR TITLE
Features/add footer top 9

### PR DESCRIPTION
**Pull request by drcases**
Write markup for footer-top.
Theme footer-top.

### CODE REVIEW

File: static/index.html [hunk 1]
```diff
@@ -63,11 +63,44 @@
...
+            <p>Drupal meetups every last Wednesday of the month, find us on <a href="https://www.meetup.com/drupal-ph/?_cookie-check=wCreirVnK3P6nhQS">meetup.com</a></p>
+            <p class="button-link">
+              <a class="button-link text-display-large" href="https://www.meetup.com/drupal-ph/?_cookie-check=wCreirVnK3P6nhQS" target="_blank">
+                Drupal Meetups every Month
+              </a>
+            </p>
+          </div>
+          <div class="block-footer-center col-md-4">
+            <h2>Drupalcebu on Twitter</h2>
+            <p><i class="fa fa-twitter" aria-hidden="true"></i> <a href="#">@DrupalCampCebu</a> We are now open for sponsorship, contact us if you would like to promote your business</p>
```
Comment (Luukyb) [inline static/index.html]: Link "DrupalCampCebu" should go to "https://twitter.com/drupalcampcebu" and open as a new tab.

File: static/index.html [hunk 2]
```diff
@@ -63,11 +63,44 @@
...
+            <p><i class="fa fa-twitter" aria-hidden="true"></i> <a href="#">@DrupalCampCebu</a> We are now open for sponsorship, contact us if you would like to promote your business</p>
```
Comment (Luukyb) [inline static/index.html]: Link "DrupalCampCebu" should go to "https://twitter.com/drupalcampcebu" and open as a new tab.

File: static/index.html [hunk 3]
```diff
@@ -63,11 +63,44 @@
...
+            <p><i class="fa fa-twitter" aria-hidden="true"></i> <a href="#">@DrupalCampCebu</a> We are now open for sponsorship, contact us if you would like to promote your business</p>
```
Comment (Luukyb) [inline static/index.html]: Link "DrupalCampCebu" should go to "https://twitter.com/drupalcampcebu" and open as a new tab.

File: static/index.html [hunk 4]
```diff
@@ -63,11 +63,44 @@
...
+          </div>
+          <div class="block-footer-right col-md-4">
+            <h2>Hosted by CIT-U</h2>
```
Comment (Luukyb) [inline static/index.html]: The university told me this week, we should use the name "CIT University".

File: static/index.html [hunk 5]
```diff
@@ -63,11 +63,44 @@
...
+            <p class="cit-details">Web: http://www.cit.edu</p>
+          </div>
+        </div>
+      </div> 
+    </div>
+    <div class="footer-bottom-wrapper">
+      <div class="container">
+        <div class="col-md-12">
+          <div class="camp-logo-footer"><img src="images/drupalcebu_puso.svg" /></div>
+          <p>Since 2014, Drupalcamp Cebu is brought to you by the <a href="https://www.facebook.com/groups/drupalcebu/">Drupal Cebu User Group.</a> <a href="#">Terms of use.</a> <a href="#">Code of conduct.</a></p>
```
Comment (Luukyb) [inline static/index.html]: Link to FB should open as a new tab. It's an external link (outside of our website).

File: static/index.html [hunk 6]
```diff
@@ -63,11 +63,44 @@
...
+          <p>Made with <i class="fa fa-heart" aria-hidden="true"></i> using Drupal 8. <a href="#">Access the source code.</a></p>
```
Comment (Luukyb) [inline static/index.html]: Link to source code is https://github.com/Luukyb/drupalcampcebu2017, and open as a new tab.

File: static/index.html [hunk 7]
```diff
@@ -63,11 +63,44 @@
...
-      <div class="col-md-12">
-        <div class="camp-logo-footer"><img src="images/drupalcebu_puso.svg" /></div>
-        <p>Since 2014, Drupalcamp Cebu is brought to you by the <a href="https://www.facebook.com/groups/drupalcebu/">Drupal Cebu User Group.</a> <a href="#">Terms of use.</a> <a href="#">Code of conduct.</a></p>
-        <p>Made with <i class="fa fa-heart" aria-hidden="true"></i> using Drupal 8. <a href="#">Access the source code.</a></p>
+    <div class="footer-top-wrapper">
+      <div class="container">
+        <div class="col-md-12">
+          <div class="block-footer-left col-md-4">
+            <h2>Drupal in Cebu</h2>
+            <p>This camp is brought to you by <a href="https://www.facebook.com/groups/drupalcebu/">Drupal Cebu User Group</a></p>
```
Comment (Luukyb) [inline static/index.html]: Link to FB should open as a new tab. It's an external link (outside of our website).

File: static/index.html [hunk 8]
```diff
@@ -63,11 +63,44 @@
...
+            <p>Access the source code of this website on Github. This website is all open source</p>
+            <p>Drupal meetups every last Wednesday of the month, find us on <a href="https://www.meetup.com/drupal-ph/?_cookie-check=wCreirVnK3P6nhQS">meetup.com</a></p>
```
Comment (Luukyb) [inline static/index.html]: Link to meetup.com should open as a new tab. It's an external link (outside of our website). Also no need for the _cookie_check. URL is only https://www.meetup.com/drupal-ph/.

File: static/index.html [hunk 9]
```diff
@@ -63,11 +63,44 @@
...
+            <p class="button-link">
+              <a class="button-link text-display-large" href="https://www.meetup.com/drupal-ph/?_cookie-check=wCreirVnK3P6nhQS" target="_blank">
```
Comment (Luukyb) [inline static/index.html]: No need for the _cookie_check. URL is only https://www.meetup.com/drupal-ph/.

File: static/index.html (hunk 2 above)
Comment (drcases) [reply static/index.html]: Done.

File: static/index.html (hunk 1 above)
Comment (drcases) [reply static/index.html]: Done.

File: static/index.html (hunk 3 above)
Comment (drcases) [reply static/index.html]: Done

File: static/index.html (hunk 4 above)
Comment (drcases) [reply static/index.html]: Done.

File: static/index.html (hunk 5 above)
Comment (drcases) [reply static/index.html]: Done.

File: static/index.html (hunk 6 above)
Comment (drcases) [reply static/index.html]: Done.

**Comment by Luukyb** [comment]
Thanks @drcases !

**Pull request merged**